scripts/mysql_system_users.sql

```diff
@@ -78,7 +78,10 @@ REVOKE ALL PRIVILEGES, GRANT OPTION FROM 'mysql.pxc.internal.session'@localhost;
 
 -- Due to bugs with roles, we need to grant superuser access here
 GRANT ALL PRIVILEGES ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
-GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER, SHUTDOWN,CLONE_ADMIN, SYSTEM_USER,GROUP_REPLICATION_STREAM  ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
+-- Needed by sst_xtrabackup
+GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
```

Comment:
If possible, please add a comment on why every listed privilege is required. You may refer to line numbers 69-73 for the already existing comments and extend it.

Reply:
Well as far as I know:
LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER -> are required by the XB process 
BACKUP_ADMIN, SHUTDOWN AND CLONE_ADMMIN are required by clone. 
However only BACKUP_ADMIN is needed by pxc.internal given the other are granted on client(receiver) during init phase so by root

Reply:
Not relevant anymore. I've remove this grant.